config/samples/nim/guardrail/nimpipeline.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: apps.nvidia.com/v1alpha1
```

Comment:
can you please change the name to guardrailpipeline?

Reply:
done